internal/interpreter/dynamic_item_tree.rs

```diff
@@ -2172,7 +2172,7 @@ extern "C" fn supported_accessibility_actions(
         .accessibility_props
         .0
         .keys()
-        .filter_map(|x| x.strip_prefix("acessible-action-"))
+        .filter_map(|x| x.strip_prefix("accessible-action-"))
```

Comment:
This one makes me shake my head - how did this work before?

Reply:
I thought there would be the counter-part but haven't found it.

Reply:
Yeah I think your change is entirely correct. This probably never worked before....

Reply:
When I test accessible actions, I use the accessibility inspector, which requires an app bundle. So I always end up compiling rust code, instead of using the interpreter. But evidently we're missing automated test coverage.

Reply:
Thanks for the confirmation.